test/integer.test.js

```diff
@@ -40,6 +40,10 @@ test('render a float as an integer', (t) => {
   const cases = [
     { input: Math.PI, output: '3' },
     { input: 5.0, output: '5' },
+    { input: null, output: '0' },
```

Comment:
Not sure this was expected because the nullable parameter, I think the output should be an empty string